core/common/src/main/java/org/phenotips/remote/common/internal/api/DefaultPatientToJSONConverter.java

```diff
@@ -112,6 +108,30 @@ public JSONObject convert(Patient patient, boolean removePrivateData, int includ
         return json;
     }
 
+    private static JSONObject contact(Patient patient) {
+        // Default contact info
+        String name = "PhenomeCentral";
+        String institution = "PhenomeCentral";
+        String href = "mailto:matchmaker@phenomecentral.org";
+
+        PatientData<String> data = patient.getData("contact");
+        if (data != null && data.isNamed()) {
+            name = data.get("name");
+            institution = data.get("institution");
```

Comment:
@allasm We do not always have an institution, but I think we should replace the default if we have owner details, even if the institution is blank.